cluster/gce/util.sh

```diff
@@ -499,28 +502,30 @@ function create-firewall-rule() {
 
 # Format the string argument for gcloud network.
 function make-gcloud-network-argument() {
-  local network="$1"
-  local address="$2"          # optional
-  local enable_ip_alias="$3"  # optional
-  local alias_subnetwork="$4" # optional
-  local alias_size="$5"       # optional
+  local network_project="$1"
+  local region="$2"
+  local network="$3"
+  local subnet="$4"
+  local address="$5"          # optional
+  local enable_ip_alias="$6"  # optional
+  local alias_size="$7"       # optional
+
+  local networkURL="projects/${network_project}/global/networks/${network}"
+  local subnetURL="projects/${network_project}/regions/${region}/subnetworks/${subnet}"
 
   local ret=""
 
   if [[ "${enable_ip_alias}" == 'true' ]]; then
     ret="--network-interface"
-    ret="${ret} network=${network}"
+    ret="${ret} network=${networkURL}"
     # If address is omitted, instance will not receive an external IP.
     ret="${ret},address=${address:-}"
-    ret="${ret},subnet=${alias_subnetwork}"
+    ret="${ret},subnet=${subnetURL}"
     ret="${ret},aliases=pods-default:${alias_size}"
     ret="${ret} --no-can-ip-forward"
   else
-    if [[ ${ENABLE_BIG_CLUSTER_SUBNETS} != "true" || (${PREEXISTING_NETWORK} = "true" && "${PREEXISTING_NETWORK_MODE}" != "custom") ]]; then
-      ret="--network ${network}"
-    else
-      ret="--subnet=${network}"
-    fi
```

Comment:
Since we retrieve the subnet in `create-subnets`, I believe we can always specify the subnet regardless of the situation.